### PR TITLE
Third f2f meeting blog

### PR DESCRIPTION
**Pull request by GZhang2**
Hi Team, 

Since I will be in Mexico till Thurs, I thought I'd better get the meeting blog done before leaving. I've prepared a draft. It is 80% done, leaving just a part for Gaurav to fill in on the curation tool. But I think what I have so far is good for review, independent of the "curation tool" part. 
I focused on the most major issues we covered during the meeting, leaving lots of the details not written down, to give focus to the blog. Gaurav's extensive meeting minutes helped a lot with drafting this blog. A special shoutout to him. 

Btw, the mark down page is here: https://github.com/phyloref/phyloref.github.io/blob/thirdf2fblog/_posts/blog/2018-03-23-Third-face-to-face-project-meeting.md

Cheers,
Guanyang

### CODE REVIEW

File: _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+## Ontology of phyloreferences
+We will prioritize our time till up to August to work on a project to transcribe the content of the new Phyloregnum, essentially a list of clade definitions, that is to be published as a book in the near future, into an ontology of phyloreferences. We plan to write a paper that would describe the ontology itself, how it works, and some illustrative examples as to what research questions can be answered with it. This paper will, however, not focus on the theory of phyloreferencing and its relevance in the broad biology research context. The intent is to publish this paper more or less simultaneously with the publication of the clade definition book. We discussed the outline of this paper, which would include a brief description of phyloreferencing, a list of phyloreferences in OWL format (deposited in a database and referred to in the paper), ontology specifications, and two use cases that will illustrate the use of the ontology of phyloreferences.
```
Comment (gaurav) [inline _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md]: - I think the book is expected to be called the Phylonyms (see https://www.ohio.edu/phylocode/preamble.html) -- that might be a better name than "the new Phyloregnum"?
- You need to provide the full form of OWL (Web Ontology Language) the first time you use it. A link to https://en.wikipedia.org/wiki/Web_Ontology_Language might also be useful!
- I'm not sure what you mean by "deposited in a database and referred to in the paper" -- please clarify? I think the plan is to stored curated phyloreferences in PHYX files, which we can later concatenate into an OWL ontology, so I'm not sure which database you're thinking of here?

File: _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+We came out with two meaningfully and potentially implementable use cases. The first is to determine relationships between phyloreferences on the original (reference) phylogeny. We ask the question "Can relationships between clade definitions be computationally determined within a given phylogeny?" We will supply a list of clade definitions and a phylogeny to the machine, and we expect it to return a list of statements about the relationships between these definitions. For example, on the phylogeny in the figure below, the computer should tell us that *Exostratum* and *Leucophanes* are disjoint and *Albifolium* includes *Exostratum* and *Leucophanes*. In other words, the computer should be able to determine that *Exostratum* and *Leucophanes* do not share descendants, but all the descendants of *Exostratum* and *Leucophanes* are also descendants of *Albifolium*. We would like to do this with an OWL reasoner. The same test can also be performed on a different phylogeny to determine if the relationship between the clades can be recovered.
```
Comment (gaurav) [inline _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md]: You need to fully cite the figure here: Tree 748 of Study 1624 on [TreeBase](https://treebase.org/treebase-web/search/study/taxa.html?id=1624&treeid=748), based on fig 10 from https://doi.org/10.1639/0007-2745(2007)110[46:POTCWA]2.0.CO;2

File: _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+![Part of a phylogeny extracted from Fisher et al.]({{ site.url }}/images/2018-03-23-f2fmeeting/Fisher_phylogeny.png "Part of a phylogeny showing three clades identified by resolving phylogenetic clade definitions on the phylogeny")
```
Comment (gaurav) [inline _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md]: I think a figure caption for this figure would look nice -- have a look at https://raw.githubusercontent.com/phyloref/phyloref.github.io/master/_posts/blog/2017-12-30-Phenoscape-hackathon.md to see how we've created those before.

File: _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
...opment to answer it. Use cases are supposed to drive software development. We can’t prioritize use cases until we have them in front of us, so ideally we have a lot more than we can deal with. Large and complex use cases can start with a futuristic “user story” that takes the form of a narrative – it does not have to be formally defined, which allows us to think through it as a group before we formalize it into a use case. There should be some evidence that the use case is broadly applicable, and not only relevant or interesting to a few people. For instance, the evidence may come from a publication where somebody poses a question that needs a computational solution. A use case is not abstract: you create a real person in the narrative that wants to do something real, such as a systematist or an evolutionary biology, and you pretend that the necessary infrastructure is already there. The next step from use cases is mock-ups or feasibility studies. How do we develop a use case? A use case has prerequisites, pre-conditions, an objective, and post-conditions (how do you know that you’ve met the objective). Below we will describe the use case we developed during the meeting as a group.
```
Comment (gaurav) [inline _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md]: I'm not sure we need this level of detail on what we learned about building use cases -- I think we could put that into a Wiki page at https://github.com/phyloref/organization/wiki, and link to that from here. I think blog readers would be more interested in the use case we developed.

File: _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
...  computer assess that? The taxon in their understanding can be reduced to a set of statements that can be individually tested against the phylogeny, e.g., “clade contains X”, “X is in the phylogeny but not contained in clade”, or “phylogeny does not contain X”. For this use case, we identified three prerequisites: a phylogeny, a clade definition, and a set of statements that describe the biologist’s understanding of a taxon (which also need to be computationally amenable). The following are the pre-conditions: (1) the clade definition should be resolvable on the phylogeny, (2) the set of statements can be matched to the phylogeny (“taxon contains X”, “taxon does not contain X”), and (3) if there is a statement that cannot be matched, then that implies that X was not sampled in this phylogeny. The post-conditions of this use case are two-fold. For each statement, we can correctly state that the statement is true, false or cannot be determined using this phylogeny. Besides, we show the results for each statement to the user. We thought this use case is specific enough to drive software development, but broadly relevant so that it should be interesting to biologists across the board.
```
Comment (gaurav) [inline _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md]: I think this paragraph would be easier to read if we clearly separate out the four components, so it looks like:
* Prerequisites:
  * ...
* Pre-conditions:
  * ...
* ...

File: _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
...ssues around traditional versus phylogeny-based monograph. We also talked about assessing congruence when taxon sampling differs between phylogenies. If two phylogenies each only contain two of the four major lineages of a taxon, say, Campanulaceae, we would not be able to determine the relationship of these two phylogenies (or the clades recovered as Campanulaceae on respective phylogenies). The second concept is "unambiguous". What do we mean when we say a phylogenetic clade definition is unambiguous? We asserted that phylogenetic clade definitions are unambiguous and can be resolved on a phylogeny unambigously. But we did discuss a scenario where a stem-based clade definition will not resolve on a phylogeny. If on a phylogeny (not the reference phylogeny) two internal specifiers are paraphyletic with respect to the external specifier, that clade definition will not resolve to a node. We discussed the relevance of "clade content". Under phyloreferencing, the clade content is not relevant. This premise will have some major implications as to how phyloreferencing will be used and what kinds of questions that it can be used to answer. We did not get to talk about these implications.
```
Comment (gaurav) [inline _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md]: If we didn't finish a particular discussion, I think you should make it clear that that isn't so much a clarifying concept as an open question :). Also, I just realized I missed a typo: "unambigously" -> "unambiguously".

File: _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+## Project organization
+In terms of project organization, we discussed mainly three issues. We planed for the upcoming Advisory Board meeting on Apr 23. We will provide a series of presentations taking up a total of 70 minutes, given by the project PIs as well as the two postdocs, to get the advisors up to speed with what we are doing. We will give an overview of the project, briefly describe the concept and method of phyloreferencing, present the curation tool and describe user stories. The second thing we discussed is our next face to face meeting. We planned to hold it in Oct in Duke, and it will be with held in conjunction the two postdocs' visit to Duke. Lastly, we briefly talked about two upcoming meetings, namely, TDWG 2018 and idigbio's Berkeley meeting. The entire team will be attending TDWG, but the attendance of idigbio's meeting will be contingent upon funding being obtained from the museum. Other organizational issues we discussed concerned how the participants of this project interact and the scope of the project.
+
+## Pending issues (aka the questions that keep Hilmar up at night)
```
Comment (gaurav) [inline _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md]: I think it sounds a bit weird to refer to these as the questions that keep Hilmar up at night. If he wants to write a blog post about that, then that's fine; as a project report, however, I think we should refer to them as "major concerns" or "unsolved challenges" or something like that.

File: _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+layout: post
+title: Third face to face project meeting
+author: gyzhang
+modified:
+excerpt: A report of our third face-to-face project meeting in Gainesville from Mar 21-23, 2018.
+tags: ["development plan", "project plan", "project meeting"]
+published: true
+comments: true
+image:
+  feature:
```
Comment (gaurav) [inline _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md]: I've now uploaded all the photos I took as well as the group photos we took on Friday to the Phyloreferencing shared Google Drive folder. Drop me an e-mail if you can't access them for any reason!

File: _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md (hunk 1 above)
Comment (GZhang2) [reply _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md]: - Changed to Phylonyms
- OWL fully spelled and linked
- Removed "deposited in a database..."

File: _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md (hunk 2 above)
Comment (GZhang2) [reply _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md]: Citation provided and article linked

File: _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md (hunk 8 above)
Comment (GZhang2) [reply _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md]: Thanks!

File: _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md (hunk 4 above)
Comment (GZhang2) [reply _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md]: This part was cut by more than half.

File: _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md (hunk 6 above)
Comment (GZhang2) [reply _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md]: Changed heading to "Some concepts"

File: _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md (hunk 7 above)
Comment (GZhang2) [reply _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md]: Changed to "unresolved challenges"

File: _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md (hunk 5 above)
Comment (GZhang2) [reply _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md]: Changed as suggested

**Comment by GZhang2** [comment]
Hi all, 

Gaurav added a section on the curation tool. This blog post draft is now complete. Can somebody please review and approve? Thanks.

@gaurav @hlapp @ncellinese

File: _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md (hunk 3 above)
Comment (GZhang2) [reply _posts/blog/2018-03-23-Third-face-to-face-project-meeting.md]: Added captions.

**Comment by GZhang2** [comment]
@hlapp ： I've deleted the three sections per your suggestion.

**Pull request merged**